apps/loggregator_test.go

```diff
@@ -113,6 +113,33 @@ var _ = Describe("loggregator", func() {
 			Expect(messages).To(ContainElement(EnvelopeContainingMessageLike("Muahaha")), "To enable the logging & metrics firehose feature, please ask your CF administrator to add the 'doppler.firehose' scope to your CF admin user.")
 		})
 	})
+
+	It(diegoUnsupportedTag+"shows container metrics", func() {
```

Comment:
I don't see why this test doesn't also apply to the Diego backend, as the cells also emit metrics about their running containers periodically.